CMakeLists.txt

```diff
@@ -18,6 +18,49 @@ option(ENABLE_DEVEL "Enable development asserts, checks, etc" OFF)
 option(ENABLE_REFCNT_DEBUG "Enable refcnt debugging" OFF)
 option(ENABLE_SHAREDPTR_DEBUG "Enable sharedptr debugging" OFF)
 set(TRYCOMPILE_SRC_DIR "${CMAKE_CURRENT_LIST_DIR}/packaging/cmake/try_compile")
+set(BUILT_WITH "")
```

Comment:
Let's include "CMAKE" here to differentiate from the mklove builds.

Reply:
Agreed.
In that case, I would also like to put the unmodified compiler ID strings from CMake in the list, instead of parsing the ID strings and generate different ID strings. I think that would be cleaner and easier to maintain...

Reply:
What does the compiler string look like?

Reply:
They are listed here: https://cmake.org/cmake/help/latest/variable/CMAKE_LANG_COMPILER_ID.html
E.g. for the GNU compiler collection it's GNU for both compilers C and C++.

Reply:
Go for it!

Reply:
It's already in https://github.com/edenhill/librdkafka/pull/2262/commits/b773fa76629c0ac6142a854720904a5a1be8556d ....